wasm_proto/front/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+		<!-- import js generated by Wasm compiler -->
+		<script type="text/javascript" src="../core/a.out.js"></script>
+
+		<!-- wrap C function in js closure -->
+		<script type="text/javascript">
+			var pc = Module.cwrap('pc', 'number', null);
+		</script>
+	</head>
+	<body>
+		<!-- call exported C function -->
+		<button type=button onclick="Module.ccall('run', null);">Run</button>
```

Comment:
I prefer [event listeners](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) because I'm not a fan of inline JS and CSS and [some things listed here](https://stackoverflow.com/questions/6348494/addeventlistener-vs-onclick). Do you have a reason for using `onclick` or can we use `addEventListener`?

btw, if you do you wanna switch over, this would be something I can address in a follow-up PR or you can address it now (your choice).